services/etl/dags/api_interface/get_waqi_data.py

```diff
@@ -0,0 +1,32 @@
+from config import get_env_configs
+import requests
+
+
+latitude = "40.04967209583848"
+longitude = "-105.28730354750316"
+
+def get_waqi_data(lat: str = latitude, long: str = longitude) -> dict:
+    """
+    TO-DO: add exception handling...better way to format dictionary?
+    """
+    get_url = get_env_configs().WAQI_BASE_URL + "feed/geo:" + latitude + ";" + longitude + "/?token=" + get_env_configs().WAQI_TOKEN
+    response = requests.get(get_url)
+    aq_json = response.json()['data']
+    result_json = {'station_name': aq_json.get('city').get('name'),
+                    'longitude': aq_json.get('city').get('geo')[0],
+                    'latitude': aq_json.get('city').get('geo')[1],
+                    'reading_datetime': aq_json.get('time').get('iso'),
+                    'request_datetime': aq_json.get('debug').get('sync'),
```

Comment:
would it be better to have the `request_datetime` come from our end via `datetime.now()` ?

Reply:
yeah let's agree to both use datetime.now() when request is made